clustered/server/src/main/java/org/ehcache/clustered/server/management/Management.java

```diff
@@ -127,26 +129,31 @@ public void clientDisconnected(ClientDescriptor clientDescriptor, ClientState cl
   public void clientReconnected(ClientDescriptor clientDescriptor, ClientState clientState) {
     if (managementRegistry != null) {
       LOGGER.trace("clientReconnected({})", clientDescriptor);
-      managementRegistry.refresh();
+      managementRegistry.refresh(); // required because ClientState fields have been modified
```

Comment:
@ljacomet : I think this is why the build was failing... These refresh() calls are needed because `ClientState` can have changed